finalPrep/modZeroHero.js

```diff
@@ -1,49 +1,92 @@
 //   catchphrase should be assigned to a string that uses interpolation to include the specialAbility
 
+var greeting = "I am " + heroName;
+var catchphrase = `I can ${specialAbility}!`;
+
 // Declare two variables - power AND energy - set to integers
+var power = 1000;
+var energy = 1000;
 
 // Declare two variables - fullPower AND fullEnergy
 //   fullPower should multiply your current power by 500
 //   fullEnergy should add 150 to your current energy
 
-// Declare two variables - isHuman and identityConcealed - assigned to booleans
+var fullPower = power * 500;
+var fullEnergy = energy + 150;
 
+// Declare two variables - isHuman and identityConcealed - assigned to booleans
+var isHuman = false;
+var identityConcealed = true;
 
 // Declare two variables - archEnemies AND sidekicks
 //   archEnemies should be an array of at least 3 different enemy strings
 //   sidekicks should be an array of at least 3 different sidekick strings
 
+var archEnemies = ["Batman", "Lex Luthor", "Kryptonite"];
+var sidekicks = ["Superboy", "Supergirl", "Beppo"];
+
 // Print the first sidekick to your console
 
+console.log(sidekicks[0]);
+
 // Print the last archEnemy to the console
 
+console.log(archEnemies[2]);
+
 // Write some code to add a new archEnemy to the archEnemies array
 
+archEnemies.push("Brainiac");
+
 // Print the archEnemies array to console to ensure you added a new archEnemy
 
+console.log(archEnemies);
+
 // Remove the first sidekick from the sidekicks array
 
+sidekicks.shift();
+
 // Print the sidekicks array to console to ensure you added a new sidekick
 
+console.log(sidekicks);
+
 // Create a function called assessSituation that takes three arguments - dangerLevel, saveTheDay, badExcuse
 //   - dangerLevel should be an integer
-//   - saveTheDay should be a string a hero would say once they save the day 
+//   - saveTheDay should be a string a hero would say once they save the day
 //   - badExcuse should be a string a hero would say if they are too afraid of the dangerLevel
 
 // Your function should include an if/else statement that meets the following criteria
 //   - Danger levels that are above 50 are too scary for your hero. Any danger level that is above 50 should result in printing the badExcuse to the console
 //   - Anything dangerLevel that is between 10 and 50 should result in printing the saveTheDay string to the console
 //   - If the dangerLevel is below 10, it means it is not worth your time and should result in printing the string "Meh. Hard pass." to the console.
 
+function assessSituation (dangerLevel, saveTheDay, badExcuse) {
+  var saveTheDay = "Job done!";
+  var badExcuse = "Is that kryptonite?";
+
+  if (dangerLevel > 50) {
+    console.log(badExcuse);
+  } else if (dangerLevel <= 50 && dangerLevel >= 10) {
+    console.log(saveTheDay);
+  } else {
+    console.log("Meh. Hard pass.");
+  }
+}
+assessSituation(99);
+assessSituation(21);
+assessSituation(3);
```

Comment:
🚨, check out lines 87-92. This is how these calls should have looked to account for what you've passed. This way your work is dynamic :)